cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -784,7 +798,7 @@ def _configure_cluster(self):
                 # Authentication is enabled, we should add this node to the
                 # pg_hba in case this is being added to an existing cluster
                 patroni_conf = self._get_patroni_dcs_conf(local_only=False)
-                node_ip = config[MANAGER][PRIVATE_IP]
+                node_ip = private_ip
```

Comment:
I note that this is just using something that was defined long ago but for some reason not used here. Does it not need any other handling, or is it just a neatness change?

Reply:
That's just a neatness/consistency thing. We're using `private_ip` a lot in between lines 695 and 788 (and don't use `config[MANAGER][PRIVATE_IP]` at all).  Just wanted to stick to that.